src/controllers/commands/builtin/twitch/UpdateChannel.cpp

```diff
@@ -40,10 +40,46 @@ QString setTitle(const CommandContext &ctx)
             channel->addMessage(
                 makeSystemMessage(QString("Updated title to %1").arg(title)));
         },
-        [channel{ctx.channel}] {
-            channel->addMessage(
-                makeSystemMessage("Title update failed! Are you "
-                                  "missing the required scope?"));
+        [channel{ctx.channel}](auto error, auto message) {
+            QString errorMessage = QString("Failed to set title - ");
+
+            using Error = HelixUpdateChannelError;
+
+            switch(error)
+            {
+                case Error::UserMissingScope: {
+                    errorMessage += "Missing required scope. "
+                                    "Re-login with your "
+                                    "account and try again.";
+                }
+                break;
+
+                case Error::UserNotAuthorized: {
+                    errorMessage += "You must be the broadcaster "
+                                    "to set the title.";
+                }
+                break;
+
+                case Error::Ratelimited: {
+                    errorMessage +=
+                        "You are being ratelimited by Twitch. Try "
+                        "again in a few seconds.";
+                }
+                break;
+
+                case Error::Forwarded: {
+                    errorMessage += message;
+                }
+                break;
+
+                case Error::Unknown:
+                default: {
+                    errorMessage += "An unknown error has occured.";
```

Comment:
This should include the message - like "Failed to set title - An unknown error has occurred ({message})" (there's also a typo in occured → occurred, which is also present in the error handling of `manageAutoModMessages` and the comment for `secondsSinceDisconnect`).